### ISSUE TITLE
test_id:1511 is failing often on k8s-1.19 lane

### ISSUE THREAD

**Issue by dhiller** [kubevirt/kubevirt#4254]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What happened**:

On all 1.19 lane fails the test [test_id:1511](https://github.com/kubevirt/kubevirt/blob/c00e33c2c0ca003ecfd50d343bbbf094ce99d8a1/tests/networkpolicy_test.go#L90) is the one to be seen most. Before making the lane 1.19 a default we need to investigate this failure.

Examples:
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4247/pull-kubevirt-e2e-k8s-1.19/1308907636502040577
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4235/pull-kubevirt-e2e-k8s-1.19/1308892285437480960
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4156/pull-kubevirt-e2e-k8s-1.19/1308846233468538880
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4051/pull-kubevirt-e2e-k8s-1.19/1308807856631320576
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4248/pull-kubevirt-e2e-k8s-1.19/1308772372039340032
* https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4235/pull-kubevirt-e2e-k8s-1.19/1308756016162148352

Output:
```
[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:component]Networkpolicy when three cirros VMs with default networking are started and serverVMI start an HTTP server on port 80 and 81 and connectivity between VMI/s is blocked by Default-deny networkpolicy [test_id:1511] should fail to reach serverVMI from clientVMI

/root/go/src/kubevirt.io/kubevirt/tests/utils.go:3008
Test Panicked
/gimme/.gimme/versions/go1.13.14.linux.amd64/src/runtime/panic.go:679

Panic: interface conversion: runtime.Object is *v1.Status, not *v1.Event

Full stack:
panic(0x1d2eca0, 0xc001f34c90)
	/gimme/.gimme/versions/go1.13.14.linux.amd64/src/runtime/panic.go:679 +0x1b2
kubevirt.io/kubevirt/tests.(*ObjectEventWatcher).Watch.func3(0x24a4be0, 0xc000d50040, 0xc001a763a0, 0xc00315de00)
	/root/go/src/kubevirt.io/kubevirt/tests/utils.go:359 +0x144
created by kubevirt.io/kubevirt/tests.(*ObjectEventWatcher).Watch
	/root/go/src/kubevirt.io/kubevirt/tests/utils.go:356 +0x513
```

**What you expected to happen**:
Test should not fail.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by dhiller** [comment]
Related kubevirt build watcher issue: https://github.com/kubevirt/kubevirt/issues/4239

**Comment by dhiller** [comment]
Closing this in favor of other issue, PR related seems ready: https://github.com/kubevirt/kubevirt/issues/4239

**Comment by jean-edouard** [comment]
I could not reproduce this issue on the latest master with 2 k8s 1.19 nodes, seems fixed indeed.
